packages/firebase_admob/ios/Classes/FLTMobileAd.m

```diff
@@ -13,11 +13,15 @@ @implementation FLTMobileAd
 NSNumber *_mobileAdId;
 FlutterMethodChannel *_channel;
 FLTMobileAdStatus _status;
+int _anchorOffset;
+int _anchorType;
```

Comment:
Can we make this an `NSLayoutYAxisAnchor`?

Reply:
Not sure how this works for iOS <11, so left it. The rest of your recommendations look good to me.